src/plugins/dashboard/public/dashboard_container/embeddable/integrations/controls/dashboard_control_group_integration.ts

```diff
@@ -181,7 +181,7 @@ function startSyncingDashboardControlGroup(this: DashboardContainer) {
         ),
         skip(1) // skip first filter output because it will have been applied in initialize
       )
-      .subscribe(() => this.forceRefresh())
+      .subscribe(() => this.forceRefresh(false)) // we should not reload the control group when the control group output changes - otherwise, performance is severely impacted
```

Comment:
Thank you for adding this comment, it will prevent us from making the mistake I made last time!